microsoft-edge/devtools-guide-chromium/javascript/breakpoints.md

```diff
@@ -230,6 +231,25 @@ When you're calling `debug()` from the DevTools Console, here's a technique to e
 1. Call `debug()` in the DevTools Console, while the code is still paused on your line-of-code breakpoint.
 
 
+<!-- ====================================================================== -->
+## Logpoints
+
+A Logpoint is a breakpoint variant that does not "break" in the debugger, but instead logs a message directly to the console.  You insert a logpoint via DevTools the same way you would any other breakpoint. 
+
+To set a logpoint:
+
+1. Select the Sources tool.
+
+1. Open the file that contains the line of code you want to insert a logpoint.
+
+1. Right-click on the left of the line of code on the line number column.
+
+1. Select **Add Logpoint**. A dialog is displayed underneath the line of code.
+
+1. Press Enter to activate the logpoint. A red diamond (or more recently, an orange icon) appears on top of the line number column:
```

Comment:
We're missing a step here. Once the dialog appears, you're supposed to enter a message or a JS expression that will get evaluated when the breakpoint is hit. Only then should you press Enter.

Reply:
Also, can you wrap the word Enter in backticks? 

```
`Enter`
instead of 
Enter
```

Reply:
@captainbrosset I think I addressed this